### PR TITLE
[minor] Add Sparse Enum support to Measurement Client

### PR DESCRIPTION
**Pull request by varshini-senthil**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md). _(Required)_
- [ ]  Automatically post PR comments with images for G code changes? _(Recommended for small changes)_

### What does this Pull Request accomplish?

- Currently, we generate `Unsupported data type` error for sparse enums, as it is not supported by LabVIEW. But measurements in Python and C# may contain sparse enums. Ex: [nidmm_measurement](https://github.com/ni/measurement-plugin-python/tree/main/examples/nidmm_measurement).
- So, updated the `Create Control.vi` to create a `Ring` control for Sparse enums and set both the item names and values to the control based on the metadata.
- Updated the `Parse JSON Enum Values.vi` to get the list of `enum items` and also `enum items and values`.
- Updated the controls in `Create Control.vi` and `Map Numeric Representation.vi` to remove coercion dots. Also, removed the `Merge Errors` node in `Create Control.vi` as it is redundant because of the standard error handling.

### Why should this Pull Request be merged?

- To provide sparse enum support to measurement client.

### What testing has been done?

- Manually tested by creating client for python measurements with sparse enums.

### CODE REVIEW

**Comment by varshini-senthil** [comment]
`Create Control.vi`
Diff image

Before:
image

After:
image

**Comment by varshini-senthil** [comment]
`Parse JSON Enum Values.vi`
Diff image

Before:
image

After:
image

File: Source/Generator/_MeasurementClient Utility/Parse JSON Enum Values.vi
Comment (dixonjoel) [inline Source/Generator/_MeasurementClient Utility/Parse JSON Enum Values.vi:1]: I don't see a reason to return the items and items and values in two different arrays. This is probably only called in a couple places, so I would remove the 'enum items' output and update callers to unbundle.

**Review by dixonjoel — APPROVED** [review]
Approved with a minor suggestion.

File: Source/Generator/_MeasurementClient Utility/Parse JSON Enum Values.vi
Comment (varshini-senthil) [reply Source/Generator/_MeasurementClient Utility/Parse JSON Enum Values.vi:1]: I've updated as per your suggestion.

Comment (varshini-senthil) [reply Source/Generator/_MeasurementClient Utility/Parse JSON Enum Values.vi:1]: The `Enum` control only expects the item names as an array of strings, while we can set both the item names and values as an array of clusters (string and value) for a `Ring` control. If we are getting only the `enum items and values` as the output from `Parse JSON Enum Values.vi`, we have to again iterate through the array of clusters, to get the array of strings for a `Enum` control.

I think using another for loop in the caller to unbundle each cluster from the `enum items and values` array to get the `enum items` is redundant, when we can get the same during the intial parsing inside `Parse JSON Enum Values.vi`. 

Would like to know your thoughts on this.

Comment (dixonjoel) [reply Source/Generator/_MeasurementClient Utility/Parse JSON Enum Values.vi:1]: Yeah that's fine. We can keep as is.

**Pull request merged**